docs/_developers/Go-bindings-to-Ethereum-contracts.md

```diff
@@ -83,7 +83,7 @@ of the contract itself, and need to specify a `backend` through which to access
 The binding generator provides out of the box an RPC backend through which you can attach
 to an existing Ethereum node via IPC, HTTP or WebSockets.
 
-We'll use the foundation's [Unicorn](https://ethereum.org/donate) token contract deployed
+We'll use the foundation's token contract deployed
```

Comment:
I think it should still state unicorn token here

Reply:
done